### PR TITLE
Update README URLs based on HTTP redirects

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein

### HTTPS Corrected URLs 
Was | Now 
--- | --- 
http://arstechnica.co.uk/security/2016/02/the-nsas-skynet-program-may-be-killing-thousands-of-innocent-people/ | https://arstechnica.co.uk/security/2016/02/the-nsas-skynet-program-may-be-killing-thousands-of-innocent-people/ 
http://cacm.acm.org/blogs/blog-cacm/169199-data-science-workflow-overview-and-challenges/fulltext | https://cacm.acm.org/blogs/blog-cacm/169199-data-science-workflow-overview-and-challenges/fulltext 
http://creativecommons.org/licenses/by/4.0/ | https://creativecommons.org/licenses/by/4.0/ 
http://datascience.stackexchange.com/ | https://datascience.stackexchange.com/ 
http://fivethirtyeight.blogs.nytimes.com/fivethirtyeights-2012-forecast/ | https://fivethirtyeight.blogs.nytimes.com/fivethirtyeights-2012-forecast/ 
http://jvns.ca/blog/2014/06/19/machine-learning-isnt-kaggle-competitions | https://jvns.ca/blog/2014/06/19/machine-learning-isnt-kaggle-competitions 
http://rayli.net/blog/data/coursera-machine-learning-review/ | https://rayli.net/blog/data/coursera-machine-learning-review/ 
http://sebastianraschka.com/faq/docs/ml-curriculum.html | https://sebastianraschka.com/faq/docs/ml-curriculum.html 
http://simplystatistics.org/2014/05/22/10-things-statistics-taught-us-about-big-data-analysis/ | https://simplystatistics.org/2014/05/22/10-things-statistics-taught-us-about-big-data-analysis/ 
http://softwareengineeringdaily.com/2015/11/17/machine-learning-and-technical-debt-with-d-sculley/ | https://softwareengineeringdaily.com/2015/11/17/machine-learning-and-technical-debt-with-d-sculley/ 
http://stats.stackexchange.com/questions/29713/what-is-covariance-in-plain-language | https://stats.stackexchange.com/questions/29713/what-is-covariance-in-plain-language 
http://stats.stackexchange.com/questions/tagged/machine-learning?sort=frequent&pageSize=15 | https://stats.stackexchange.com/questions/tagged/machine-learning?sort=frequent&pageSize=15 
http://trendct.org/2016/08/05/real-world-data-cleanup-with-python-and-pandas/ | https://trendct.org/2016/08/05/real-world-data-cleanup-with-python-and-pandas/ 
http://www.datascienceweekly.org/data-science-resources/data-science-moocs | https://www.datascienceweekly.org/data-science-resources/data-science-moocs 
http://www.datascienceweekly.org/data-science-resources/the-big-list-of-data-science-resources | https://www.datascienceweekly.org/data-science-resources/the-big-list-of-data-science-resources 
http://www.forbes.com/sites/anthonykosner/2013/12/29/why-is-machine-learning-cs-229-the-most-popular-course-at-stanford/ | https://www.forbes.com/sites/anthonykosner/2013/12/29/why-is-machine-learning-cs-229-the-most-popular-course-at-stanford/ 
http://www.kaggle.com/competitions | https://www.kaggle.com/competitions 
http://www.tamr.com/landing-pages/getting-data-right/ | https://www.tamr.com/landing-pages/getting-data-right/ 
http://www.tensorflow.org/ | https://www.tensorflow.org/ 

### Other Corrected URLs 
Was | Now 
--- | --- 
http://c2.com/cgi/wiki?DoSimpleThings | http://wiki.c2.com/?DoSimpleThings 
http://conda.pydata.org/docs/_downloads/conda-pip-virtualenv-translator.html | https://conda.io/docs/_downloads/conda-pip-virtualenv-translator.html 
http://nbviewer.ipython.org/github/ClickSecurity/data_hacking/blob/master/dga_detection/DGA_Domain_Detection.ipynb | http://nbviewer.jupyter.org/github/ClickSecurity/data_hacking/blob/master/dga_detection/DGA_Domain_Detection.ipynb 
http://nbviewer.ipython.org/github/ClickSecurity/data_hacking/blob/master/java_classification/java_classification.ipynb | http://nbviewer.jupyter.org/github/ClickSecurity/data_hacking/blob/master/java_classification/java_classification.ipynb 
http://nbviewer.ipython.org/github/ClickSecurity/data_hacking/blob/master/sql_injection/sql_injection.ipynb | http://nbviewer.jupyter.org/github/ClickSecurity/data_hacking/blob/master/sql_injection/sql_injection.ipynb 
http://nbviewer.ipython.org/github/jmsteinw/Notebooks/blob/master/IndeedJobs.ipynb | http://nbviewer.jupyter.org/github/jmsteinw/Notebooks/blob/master/IndeedJobs.ipynb 
http://nbviewer.ipython.org/github/ogrisel/notebooks/blob/master/Labeled%20Faces%20in%20the%20Wild%20recognition.ipynb | http://nbviewer.jupyter.org/github/ogrisel/notebooks/blob/master/Labeled%20Faces%20in%20the%20Wild%20recognition.ipynb 
http://nbviewer.ipython.org/github/rasbt/python_reference/blob/master/tutorials/things_in_pandas.ipynb | http://nbviewer.jupyter.org/github/rasbt/python_reference/blob/master/tutorials/things_in_pandas.ipynb 
http://www.jacksimpson.co/materials-for-learning-machine-learning/ | http://jacksimpson.co/materials-for-learning-machine-learning/ 
https://docs.google.com/document/d/1YN6BVdReNAYc8B0fjQ84yzDflqmeEPj7S0Xc-9_26R0/ | https://docs.google.com/document/d/1YN6BVdReNAYc8B0fjQ84yzDflqmeEPj7S0Xc-9_26R0/edit 
https://drive.google.com/folderview?id=0ByIrJAE4KMTtaGhRcXkxNHhmY2M | https://drive.google.com/drive/folders/0ByIrJAE4KMTtaGhRcXkxNHhmY2M 
https://medium.com/machine-learnings/a-humans-guide-to-machine-learning-e179f43b67a0 | https://machinelearnings.co/a-humans-guide-to-machine-learning-e179f43b67a0 
https://www.coursera.org/specializations/jhudatascience | https://www.coursera.org/specializations/jhu-data-science

### CODE REVIEW

**Comment by hangtwenty** [comment]
@ReadmeCritic Is there a way to get this to fire again on-demand? I missed the initial notification but the automated changes seem very nifty, would like to ensure it's up to date

**Comment by ReadmeCritic** [comment]
no there is no automatic way to launch this.. sorry

**Comment by ReadmeCritic** [comment]
for this pull request, is there a problem ? pls let me know

**Comment by hangtwenty** [comment]
@ReadmeCritic Well the Travis CI is failing on this PR... This Travis CI is using `awesome_bot` to check the links, I remember that both `awesome_bot` and @ReadmeCritic are by @dkhamsing 😄 so just wondering which thing takes precedence. But, looking at the failure, it's actually just 1 URL that is failing, and it's not due to this PR. I know what to do now, thanks!

**Comment by hangtwenty** [comment]
I fixed that one issue in master, did a little revision there too, and accidentally caused a merge conflict. But it's easy to resolve! So I'll do that now. Thanks again.

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ I suggest you get your feet wet ASAP. You'll boost your confidence.
 
 You can install Python 3 and all of these packages in a few clicks with the [Anaconda Python distribution](https://www.continuum.io/downloads). Anaconda is popular in Data Science and Machine Learning communities.
 
-If you're using Python 2.7, don't worry. You don't have to migrate to Python 3 just for this guide. Also, if you're using pip/virtualenv instead of Anaconda, that's alright too! Cf. [conda vs. pip vs. virtualenv](http://conda.pydata.org/docs/_downloads/conda-pip-virtualenv-translator.html) if you're curious.
+If you're using Python 2.7, don't worry. You don't have to migrate to Python 3 just for this guide. Also, if you're using pip/virtualenv instead of Anaconda, that's alright too! Cf. [conda vs. pip vs. virtualenv](https://conda.io/docs/_downloads/conda-pip-virtualenv-translator.html) if you're curious.
```
Comment (hangtwenty) [inline README.md]: Note to self, I must update this one after merging, URL has actually changed further to this:

https://conda.io/docs/commands.html#conda-vs-pip-vs-virtualenv-commands

File: README.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ So  **knowledge** and **data** are critical. Focus your efforts on those, before
 
 (Chart inspired by a slide from [Alex Pinto's talk, "Secure Because Math: A Deep-Dive on ML-Based Monitoring"](https://www.youtube.com/watch?v=TYVCVzEJhhQ).)
 
-[Do Simple Things]: http://c2.com/cgi/wiki?DoSimpleThings
+[Do Simple Things]: http://wiki.c2.com/?DoSimpleThings
```
Comment (hangtwenty) [inline README.md]: Confirmed this new URL is active and correct

File: README.md [hunk 3]
```diff
@@ -95,14 +95,14 @@ OK. Take a break, come back fresh.
 
 Next, pick **one or two** of these IPython Notebooks and play along.
 
-- [Face Recognition on a subset of the Labeled Faces in the Wild](http://nbviewer.ipython.org/github/ogrisel/notebooks/blob/master/Labeled%20Faces%20in%20the%20Wild%20recognition.ipynb)
+- [Face Recognition on a subset of the Labeled Faces in the Wild](http://nbviewer.jupyter.org/github/ogrisel/notebooks/blob/master/Labeled%20Faces%20in%20the%20Wild%20recognition.ipynb)
```
Comment (hangtwenty) [inline README.md]: Note to self, this one is not right, I will fix it in master after merging. Correct =

http://nbviewer.ipython.org/github/ogrisel/notebooks/blob/master/Labeled%20Faces%20in%20the%20Wild%20recognition.ipynb

**Pull request merged**